app/models/entry.server.ts

```diff
@@ -142,3 +143,32 @@ export function getEntriesByBasicTextSearch(
       LOWER(headword) LIKE LOWER(${searchWildcard}))  
     ORDER BY headword ASC LIMIT ${take} OFFSET ${skip}`
 }
+
+export async function updateRecordByAttributeAndType(
+  type: number, // probably change this to enum
```

Comment:
Yes, this will need to change to the type of the enum.

Reply:
done!